src/content/en/updates/2018/09/reportingapi.md

```diff
@@ -74,7 +74,7 @@ to report errors to:
 Report-To: {
              "max_age": 10886400,
              "endpoints": [{
-               "url": "https://analytics.provider.com/browser-errors"
+               "url": "https://example.com/browser-errors"
```

Comment:
please leave as is.

Reply:
That makes the text that follows incorrect: "In the example, sending this response header with your main page configures the browser to report browser-generated warnings to the endpoint https://example.com/browser-errors for max_age seconds."